contracts/commands/BasicBuyCommand.sol

```diff
@@ -69,7 +71,7 @@ contract BasicBuyCommand is BaseMPACommand {
             trigger.deviation
         );
         return
-            trigger.triggerType == 3 &&
```

Comment:
I think we should leave original cammands with their types as they are and new ones with new types, and link tests only to new ones